go/src/koding/kites/kloud/kloud/kloud.go

```diff
@@ -462,118 +411,7 @@ func newSession(conf *Config, k *kite.Kite) (*session.Session, error) {
 	return sess, nil
 }
 
-// Providers ctors. If a provider has a dependencty on a service
-// defined in *session.Session, and the service is disabled (nil)
-// due to missing configuration or any other reason, we return
-// disabled provider that rejects all requests with 411 *kite.Error.
-
-func newKodingProvider(sess *session.Session, conf *Config, authUsers map[string]string) stack.Provider {
```

Comment:
Let's merge https://github.com/koding/koding/pull/9042 first and then I'll rebase my PR.